diff --git a/platforms/common/include/px4_platform_common/defines.h b/platforms/common/include/px4_platform_common/defines.h
index 38d7b4201f..6c7fb2abcd 100644
--- a/platforms/common/include/px4_platform_common/defines.h
+++ b/platforms/common/include/px4_platform_common/defines.h
@@ -39,8 +39,8 @@
 
 #pragma once
 
-#include <sys/ioctl.h>
-#include <px4_boardconfig.h>
+//#include <sys/ioctl.h>
+//#include <px4_boardconfig.h>
 
 
 /****************************************************************************
diff --git a/src/lib/geo/geo.h b/src/lib/geo/geo.h
index 30cc10b6fe..7ea7acafbe 100644
--- a/src/lib/geo/geo.h
+++ b/src/lib/geo/geo.h
@@ -45,13 +45,16 @@
 
 #pragma once
 
+#include "ecl.h"
+
+
 #include <stdbool.h>
 #include <stdint.h>
 
-#include <drivers/drv_hrt.h>
+//#include <drivers/drv_hrt.h>
 
-#include <lib/mathlib/mathlib.h>
-#include <lib/matrix/matrix/math.hpp>
+#include <mathlib/mathlib.h>
+#include <matrix/math.hpp>
 
 static constexpr float CONSTANTS_ONE_G = 9.80665f;						// m/s^2
 static constexpr double CONSTANTS_RADIUS_OF_EARTH = 6371000;					// meters (m)
diff --git a/src/lib/mathlib/math/Functions.hpp b/src/lib/mathlib/math/Functions.hpp
index 7fb54442d9..89adf7bcb2 100644
--- a/src/lib/mathlib/math/Functions.hpp
+++ b/src/lib/mathlib/math/Functions.hpp
@@ -42,7 +42,7 @@
 #include "Limits.hpp"
 
 #include <px4_platform_common/defines.h>
-#include <matrix/matrix/math.hpp>
+#include <matrix/math.hpp>
 
 namespace math
 {
diff --git a/src/modules/ekf2/EKF/aid_sources/EstimatorAidSource.hpp b/src/modules/ekf2/EKF/aid_sources/EstimatorAidSource.hpp
index 6d6c9e8ec2..55c45932cf 100644
--- a/src/modules/ekf2/EKF/aid_sources/EstimatorAidSource.hpp
+++ b/src/modules/ekf2/EKF/aid_sources/EstimatorAidSource.hpp
@@ -36,7 +36,7 @@
 
 #include <cstdint>
 
-#include <lib/matrix/matrix/math.hpp>
+#include <matrix/math.hpp>
 
 // forward declarations
 class Ekf;
diff --git a/src/modules/ekf2/EKF/drag_fusion.cpp b/src/modules/ekf2/EKF/drag_fusion.cpp
index 5b4baedf12..6176646b6b 100644
--- a/src/modules/ekf2/EKF/drag_fusion.cpp
+++ b/src/modules/ekf2/EKF/drag_fusion.cpp
@@ -41,7 +41,7 @@
 #include <ekf_derivation/generated/compute_drag_y_innov_var_and_h.h>
 
 #include <mathlib/mathlib.h>
-#include <lib/atmosphere/atmosphere.h>
+#include <atmosphere/atmosphere.h>
 
 void Ekf::controlDragFusion(const imuSample &imu_delayed)
 {
diff --git a/src/modules/ekf2/EKF/ekf_helper.cpp b/src/modules/ekf2/EKF/ekf_helper.cpp
index afff6dc0ca..1feff04e3c 100644
--- a/src/modules/ekf2/EKF/ekf_helper.cpp
+++ b/src/modules/ekf2/EKF/ekf_helper.cpp
@@ -42,7 +42,7 @@
 #include "ekf.h"
 
 #include <mathlib/mathlib.h>
-#include <lib/world_magnetic_model/geo_mag_declination.h>
+#include <world_magnetic_model/geo_mag_declination.h>
 #include <cstdlib>
 
 bool Ekf::isHeightResetRequired() const
diff --git a/src/modules/ekf2/EKF/estimator_interface.h b/src/modules/ekf2/EKF/estimator_interface.h
index e94d14b5e6..117380030f 100644
--- a/src/modules/ekf2/EKF/estimator_interface.h
+++ b/src/modules/ekf2/EKF/estimator_interface.h
@@ -71,7 +71,7 @@
 # include "sensor_range_finder.hpp"
 #endif // CONFIG_EKF2_RANGE_FINDER
 
-#include <lib/atmosphere/atmosphere.h>
+#include <atmosphere/atmosphere.h>
 #include <matrix/math.hpp>
 #include <mathlib/mathlib.h>
 #include <mathlib/math/filter/AlphaFilter.hpp>
diff --git a/src/modules/ekf2/EKF/gps_checks.cpp b/src/modules/ekf2/EKF/gps_checks.cpp
index 71018df1c7..66634a3b5d 100644
--- a/src/modules/ekf2/EKF/gps_checks.cpp
+++ b/src/modules/ekf2/EKF/gps_checks.cpp
@@ -42,7 +42,7 @@
 #include "ekf.h"
 
 #if defined(CONFIG_EKF2_MAGNETOMETER)
-# include <lib/world_magnetic_model/geo_mag_declination.h>
+# include <world_magnetic_model/geo_mag_declination.h>
 #endif // CONFIG_EKF2_MAGNETOMETER
 
 #include <mathlib/mathlib.h>
diff --git a/src/modules/ekf2/EKF/imu_down_sampler.cpp b/src/modules/ekf2/EKF/imu_down_sampler.cpp
index 02f43a12f5..10c475ffe2 100644
--- a/src/modules/ekf2/EKF/imu_down_sampler.cpp
+++ b/src/modules/ekf2/EKF/imu_down_sampler.cpp
@@ -1,6 +1,6 @@
 #include "imu_down_sampler.hpp"
 
-#include <lib/mathlib/mathlib.h>
+#include <mathlib/mathlib.h>
 
 ImuDownSampler::ImuDownSampler(int32_t &target_dt_us) : _target_dt_us(target_dt_us)
 {
diff --git a/src/modules/ekf2/EKF/output_predictor.h b/src/modules/ekf2/EKF/output_predictor.h
index ee5cc0f710..1b7fb44d74 100644
--- a/src/modules/ekf2/EKF/output_predictor.h
+++ b/src/modules/ekf2/EKF/output_predictor.h
@@ -39,7 +39,7 @@
 #include "common.h"
 #include "RingBuffer.h"
 
-#include <lib/geo/geo.h>
+#include <geo/geo.h>
 
 class OutputPredictor
 {
diff --git a/src/modules/ekf2/EKF/yaw_estimator/EKFGSF_yaw.cpp b/src/modules/ekf2/EKF/yaw_estimator/EKFGSF_yaw.cpp
index c8655827be..d465cd2273 100644
--- a/src/modules/ekf2/EKF/yaw_estimator/EKFGSF_yaw.cpp
+++ b/src/modules/ekf2/EKF/yaw_estimator/EKFGSF_yaw.cpp
@@ -35,7 +35,7 @@
 
 #include <cstdlib>
 
-#include <lib/geo/geo.h> // CONSTANTS_ONE_G
+#include <geo/geo.h> // CONSTANTS_ONE_G
 
 #include "derivation/generated/yaw_est_predict_covariance.h"
 #include "derivation/generated/yaw_est_compute_measurement_update.h"
diff --git a/src/modules/ekf2/EKF/yaw_estimator/EKFGSF_yaw.h b/src/modules/ekf2/EKF/yaw_estimator/EKFGSF_yaw.h
index ec84ed06f9..37c6fb7842 100644
--- a/src/modules/ekf2/EKF/yaw_estimator/EKFGSF_yaw.h
+++ b/src/modules/ekf2/EKF/yaw_estimator/EKFGSF_yaw.h
@@ -34,8 +34,8 @@
 #ifndef EKF_EKFGSF_YAW_H
 #define EKF_EKFGSF_YAW_H
 
-#include <lib/mathlib/mathlib.h>
-#include <lib/matrix/matrix/math.hpp>
+#include <mathlib/mathlib.h>
+#include <matrix/math.hpp>
 
 static constexpr uint8_t N_MODELS_EKFGSF = 5;
 
